src/components/AutoSuggest/SuggestList.js

```diff
@@ -0,0 +1,96 @@
+import React, { Fragment, useEffect, useRef, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { themeColor, themeSpacing, Icon } from '@datapunt/asc-ui';
+import { ChevronRight } from '@datapunt/asc-assets';
+
+const StyledList = styled.ul`
+  border: 1px solid ${themeColor('tint', 'level5')};
+  border-top: 0;
+  padding: 0;
+  margin: 0;
+`;
+
+const Li = styled.li`
+  line-height: 20px;
+  padding: 10px 20px;
+  cursor: pointer;
+  display: flex;
+
+  &:hover {
+    background-color: ${themeColor('tint', 'level2')};
+  }
+`;
+
+const Chevron = styled(ChevronRight)`
+  display: inline-block;
+`;
+
+const StyledIcon = styled(Icon)`
+  margin: 0 ${themeSpacing(2)} 0 0;
+  display: inline-block;
+`;
+
+const SuggestList = ({ activeIndex, className, role, options, onSelectOption, ...rest }) => {
+  const listRef = useRef(null);
+
+  useEffect(() => {
+    const list = listRef.current;
+
+    if (activeIndex >= 0 && activeIndex < options.length) {
+      list.children[activeIndex].focus();
+    }
+  }, [activeIndex, options.length]);
+
+  const onSelect = useCallback(
+    option => {
+      onSelectOption(option);
+    },
+    [onSelectOption]
+  );
+
+  return (
+    <StyledList className={className} role={role} ref={listRef} {...rest}>
+      {options.map(option => (
+        <Li
```

Comment:
Can we use here the styled compoennts version of the list?
`https://amsterdam.github.io/amsterdam-styled-components/?path=/story/experimental-atoms-list--default-style-with-chevron-links`

Reply:
I tried, but the styling is different from the design system and I wasn't able to get a ref to the list so that I could get the keyboard navigation right as well as the focus 🤷‍♂